ci/playbooks/edpm/run.yml

```diff
@@ -11,6 +11,6 @@
           -e @scenarios/centos-9/edpm_ci.yml
           -e @scenarios/centos-9/zuul_inventory.yml
           {%- if make_edpm_deploy_env is defined %}
-          -e "make_edpm_deploy_env={{ make_edpm_deploy_env }}"
+          -e "{'make_edpm_deploy_env':{{ make_edpm_deploy_env | from_yaml }}}"
```

Comment:
I'm wondering if we shouldn't use a plain file at this point. Passing some yaml/json may lead to really, really unwanted behavior at this point.

What if the "previous step" is generating a plain file, and we allow to inject it on-demand, for instance as proposed in #189 ?